packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -23,54 +23,71 @@ class TestCommand extends FlutterCommand {
+    argParser.addOption(
+      'name',
       help: 'A regular expression matching substrings of the names of tests to run.',
       valueHelp: 'regexp',
       allowMultiple: true,
       splitCommas: false,
     );
-    argParser.addOption('plain-name',
+    argParser.addOption(
+      'plain-name',
       help: 'A plain-text substring of the names of tests to run.',
       valueHelp: 'substring',
       allowMultiple: true,
       splitCommas: false,
     );
-    argParser.addFlag('start-paused',
-        defaultsTo: false,
-        negatable: false,
-        help: 'Start in a paused mode and wait for a debugger to connect.\n'
-              'You must specify a single test file to run, explicitly.\n'
-              'Instructions for connecting with a debugger and printed to the\n'
-              'console once the test has started.'
+    argParser.addFlag(
+      'start-paused',
+      defaultsTo: false,
+      negatable: false,
+      help: 'Start in a paused mode and wait for a debugger to connect.\n'
+            'You must specify a single test file to run, explicitly.\n'
+            'Instructions for connecting with a debugger and printed to the\n'
+            'console once the test has started.',
     );
-    argParser.addFlag('coverage',
+    argParser.addFlag(
+      'coverage',
       defaultsTo: false,
       negatable: false,
-      help: 'Whether to collect coverage information.'
+      help: 'Whether to collect coverage information.',
     );
-    argParser.addFlag('merge-coverage',
+    argParser.addFlag(
+      'merge-coverage',
       defaultsTo: false,
       negatable: false,
       help: 'Whether to merge converage data with "coverage/lcov.base.info".\n'
-            'Implies collecting coverage data. (Requires lcov)'
+            'Implies collecting coverage data. (Requires lcov)',
     );
-    argParser.addFlag('ipv6',
-        negatable: false,
-        hide: true,
-        help: 'Whether to use IPv6 for the test harness server socket.'
+    argParser.addFlag(
+      'ipv6',
+      negatable: false,
+      hide: true,
+      help: 'Whether to use IPv6 for the test harness server socket.',
     );
-    argParser.addOption('coverage-path',
+    argParser.addOption(
+      'coverage-path',
       defaultsTo: 'coverage/lcov.info',
-      help: 'Where to store coverage information (if coverage is enabled).'
+      help: 'Where to store coverage information (if coverage is enabled).',
+    );
+    argParser.addFlag(
+      'machine',
+      hide: !verboseHelp,
+      negatable: false,
+      help: 'Handle machine structured JSON command input\n'
+            'and provide output and progress in machine friendly format.',
+    );
+    argParser.addFlag(
+      'preview-dart-2',
+      hide: !verboseHelp,
+      help: 'Preview Dart 2.0 functionality.',
+    );
+    argParser.addFlag(
+      'track-widget-creation',
+      negatable: false,
+      hide: !verboseHelp,
+      help: 'Track widget creation locations.',
```

Comment:
maybe add some comment like "This enables testing of features such as the widget inspector."

Reply:
done